web/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "share-seller",
   "version": "0.0.0",
   "scripts": {
+    "start": "vite start",
```

Comment:
and why do you have a start script here, use the default `dev`, and it doesn't need further mods